meme/public/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name:  <input type="text" id="Name" />
+    Comment: <input type="text" id="Comment" />
+    Submit: <input type="submit" id="button" />
+    <br>
+    click here to see : <input type="submit" value="click picture" id="click" />
+    </br>
+    <div>
+      <pre id="chatid"></pre>
+      <video id="v1" width = "500" height = "500" ></video>
+      <canvas width= 500 height= 500 id="c1"></canvas>
+    </div>
+    <script>
+const button=document.getElementById("button");
+button.onclick = ()=>{
+  const name =document.getElementById("Name").value;
+  const comment =document.getElementById("Comment").value;
+  const xhttp=new XMLHttpRequest();
+  xhttp.open('post','/newchat');
+  xhttp.setRequestHeader('content-type','application/json');
+  xhttp.send(JSON.stringify({name,comment}));
+}
+//*displaying all chat files *//
+const startchat =() => {
+  const xhttp = new XMLHttpRequest();
+  xhttp.open('GET','/data.txt');
+  xhttp.onreadystatechange = () =>{
+    if (xhttp.responseText){
+      const chat=document.getElementById("chatid");
+      chat.innerHTML=xhttp.responseText;
+    }
+  }
+  xhttp.send();
+}
+//*camera*//
+const video = document.getElementById("v1");
+navigator.mediaDevices.getUserMedia({
+  video:true,
+  audio:false
+}).then((mediaStream) =>{
+  video.srcObject =mediaStream;
+  video.play();
+}).catch((err) => {
+  console.log("err");
+});
+//* taking a picture *//
+const button2 =document.getElementById("click");
+button2.onclick=()=>{
+  const name =document.getElementById("Name").value;
```

Comment:
you should define variables right before you use them. Therefore, move line 55+56 to line 62